backends-velox/src/test/scala/org/apache/gluten/execution/ScalarFunctionsValidateSuite.scala

```diff
@@ -978,6 +978,23 @@ class ScalarFunctionsValidateSuite extends FunctionsValidateTest {
     }
   }
 
+  testWithSpecifiedSparkVersion("get", Some("3.4")) {
```

Comment:
A little question, why we need to check it for only spark 3.4?

Reply:
this function is added since 3.4.0 https://spark.apache.org/docs/latest/api/sql/#get

Reply:
and `Some(3.4)` in above code is min spark version need to run.

Reply:
Got it, thanks.